tests/config/config_for_integration_tests.yaml

```diff
@@ -119,17 +119,6 @@ ols_config:
       - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
       - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
     minTLSVersion: VersionTLS13
-mcp_servers:
-  - name: foo
-    transport: stdio
-    stdio:
-      command: python
-      args:
-        - mcp_server_1.py
-  - name: bar
-    transport: sse
-    sse:
-      url: 127.0.0.1:8080
```

Comment:
I've remove the mcp_servers from unit/integration configs. Those are not real servers so the mcp client hangs/timeouts trying to connect.